pkg/spec/createconfig.go

```diff
@@ -403,6 +404,32 @@ func (c *CreateConfig) getContainerCreateOptions(runtime *libpod.Runtime, pod *l
 		options = append(options, libpod.WithHealthCheck(c.HealthCheck))
 		logrus.Debugf("New container has a health check")
 	}
+
+	// Add ports
+	if !c.Network.NetMode.IsHost() {
+		var ports []ocicni.PortMapping
+		for i := range c.Network.PortBindings {
```

Comment:
I'm not a fan of one char variables that are used more than a line or two away.  Suggest:
```suggestion
		for binding := range c.Network.PortBindings {
```
or a longer variable name of your choosing